src/index.html

```diff
@@ -7,6 +7,345 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <!-- #region moyo_header -->
+    <header class="header">
+      <a class="logo" href="#">
+        <img src="images/logo.png" alt="logo">
+      </a>
+      <nav class="menu">
```

Comment:
Please check for all code
![image](https://github.com/mate-academy/layout_catalog/assets/54936044/2f0b37db-dde1-439c-bc57-898a8a455d9a)
```suggestion
      <a class="logo" href="#">
        <img src="images/logo.png" alt="logo">
      </a>
      
      <nav class="menu">
```